### PR TITLE
[SPS-233] | Add location and calendar icons

### PR DESCRIPTION
**Pull request by yxshu11**
## Description

Adding 4 new SVG icons for booking services
- Location
- Location (White)
- Calendar
- Calendar (White)

## Related issue / work / context

https://sephora-asia.atlassian.net/browse/SPS-223

## Screenshots (if appropriate):
Screenshot 2020-03-17 at 11 47 54 AM

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: app/assets/images/sephora_style_guide/svgs/booking_services/calendar-white.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M14.5 0a.5.5 0 01.5.5V2h2a1 1 0 011 1v14a1 1 0 01-1 1H1a1 1 0 01-1-1V3a1 1 0 011-1h2V.5a.5.5 0 011 0V2h10V.5a.5.5 0 01.5-.5zM1 7v10h16V7H1zm0-1h16V3h-2v.5a.5.5 0 11-1 0V3H4v.5a.5.5 0 01-1 0V3H1v3z" id="a"/></defs><g transform="translate(3 3)" fill="none" fill-rule="evenodd"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><use fill="#000" xlink:href="#a"/><g mask="url(#b)" fill="#FFF"><path d="M-3-3H97V97H-3z"/></g></g></svg>
```
Comment (zubairshams) [inline app/assets/images/sephora_style_guide/svgs/booking_services/calendar-white.svg]: Should format into mull-tine for better readability.

File: app/views/sephora_style_guide/home/_icons.slim [hunk 2]
```diff
@@ -960,6 +960,26 @@ ruby:
       "size": "icon_90x40"
     }
   ]
+  booking_services_icon = [
+    {
```
Comment (zubairshams) [inline app/views/sephora_style_guide/home/_icons.slim]: Could you group together existing booking service icon here e.g `store-white/store-black`.  There might be some others.

Comment (yxshu11) [reply app/views/sephora_style_guide/home/_icons.slim]: Moved store-white, store-black and location into the booking service section

Comment (zubairshams) [reply app/views/sephora_style_guide/home/_icons.slim]: Where is location icon is being used in booking services?

**Pull request merged**